src/Pagination/Pagination.component.tsx

```diff
@@ -0,0 +1,268 @@
+// VENDOR
+import * as React from 'react';
+import classNames from 'classnames';
+import styled from '@xstyled/styled-components';
+// ANCHOR
+import { ButtonProps } from '../Button/Button.component';
+import { Goto } from './Goto';
+import {
+    Button as AnchorButton,
+    ChevronLeft,
+    ChevronRight,
+    Ellipses,
+    Typography,
+} from '..';
+
+// Constrain a value between a min and max
```

Comment:
It'd be beneficial to explain why. I feel like the code already demonstrates what is going on, but not why it's necessary.